### PR TITLE
LPS-201181 Fragment for Video Streaming that loads video.js player

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
Issue: https://liferay.atlassian.net/browse/LPS-201181 

**How to test it:** 

1. Enable Feature Flag `feature.flag.LPS-200108`
2. Go to Home Page and click on edit icon
3. In Fragments section, select **Video Streaming** form Content Display collection
4. Edit the url and add a video from https://gist.github.com/SeunghoonBaek/f35e0fd3db80bf55c2707cae5d0f7184 (these are examples, as we don't have videos on our CDNs yet)

A video player should be rendered and you can play the video you selected.

### CODE REVIEW

**Comment by ambrinchaudhary** [comment]
ci:test:sf

File: modules/apps/fragment/fragment-video-streaming/src/main/resources/META-INF/resources/page.jsp [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	<div class="videojs-container">
+		<video class="video-js" controls id="fragmentVideoJsURL" preload="auto">
+			<source src="<%= (String)request.getAttribute("src") %>" type="video/mp4" />
+		</video>
+
+		<script src="https://vjs.zencdn.net/8.6.1/video.min.js"></script>
+	</div>
+</div>
+
+<script>
```
Comment (ambrinchaudhary) [inline modules/apps/fragment/fragment-video-streaming/src/main/resources/META-INF/resources/page.jsp:26]: I know this should be in a different .js file, but it was throwing errors, and as this is a POC, for now I decided to leave it this way and keep focusing on the needs for the RFP.

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Review by marcogalluzzi — APPROVED** [review]
I've fixed myself some missing things and JSP errors that prevent the parameters to be used by the video-js component

**Comment by marcogalluzzi** [comment]
ci:forward

**Comment by marcogalluzzi** [comment]
ci:test:sf

**Comment by marcogalluzzi** [comment]
ci:forward

**Comment by WYZ-Issac** [comment]
ci:test:stable

**Comment by WYZ-Issac** [comment]
Failures are unrelated.

**Comment by WYZ-Issac** [comment]
ci:forward:force

**Pull request closed without merging**